ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -337,25 +331,24 @@ class SimpleOperationTracker implements OperationTracker {
 
     int numActiveReplicasInOriginatingDc =
         getEligibleReplicas(originatingDcName, EnumSet.of(ReplicaState.STANDBY, ReplicaState.LEADER)).size();
-    if (routerConfig.routerOperationTrackerTerminateOnNotFoundEnabled && numReplicasInOriginatingDc > 0
+    if (routerConfig.routerOperationTrackerTerminateOnNotFoundEnabled
```

Comment:
If routerOperationTrackerTerminateOnNotFoundEnabled, with the logic, it'll fix the 503 error.
But what about it's not enabled? Will we still see 503 response?